terraform/globals.tf

```diff
@@ -1,3 +1,7 @@
+terraform {
+  required_version = "0.8.7"
+}
```

Comment:
`>= 0.8.5` referenced in `version_constraint.tf` isn't correct as state file was created with version `0.8.7`. As a simplest thing we can change the version in this file to `=0.8.7` that will solve all the problems but still will be a bit confusing about plugin version. That can be refactored during terraform upgrade.